File: jss_examples/loop_call.js

function calc_sum(a, b) {
    return a + b;
};

function sum_numbers(N) {
    sum = 0;
    i = 0;
    while (i < N) {
        sum = calc_sum(sum, i);
        i = i + 1;
    }
    return sum;
};

x = 3000;
x = 30000000;
print(sum_numbers(x));

File: examples/loop_call.py

def sum_fn(a, b):
    return a + b

def sum_numbers(N):
    s = 0.0
    i = 0.0
    while i < N:
        s = sum_fn(s, i)
        i += 1.0
    return s


print sum_numbers(30000000)
